### DISCUSSION TITLE
Alt+F doesn't work in euporie-notebook (only in kitty)

### DISCUSSION THREAD

**Discussion by raffaelemancuso** [kovidgoyal/kitty#6376] [Q&A]
I use euporie-notebook.

With Konsole, ALT+F opens the "File" menu.

But this doesn't work in Kitty. ALT+F does nothing.

Why is that?

EDIT: it works also in WezTerm. And I've tried removing my custom kitty configuration (and using the default one) and nothing changed.

EDIT2: I've remove `$HOME/.config/kitty` entirely and still doesn't work

**Comment by kovidgoyal** [comment]
kitty doesnt have a file manu. If you mena a file menu inside euporie,
then you will need to ask its developers.

**Reply by raffaelemancuso** [reply]
Yes I mean the file menu inside euporie.

It seems to be a bug in euporie. See [here](https://github.com/joouha/euporie/issues/83#issuecomment-1597145187).
